library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/impl/reloader/AttachmentDictionary.java

```diff
@@ -40,22 +40,20 @@
 final class AttachmentDictionary<R, V> {
 	private final Registry<R> registry;
 	private final RegistryEntryAttachment<R, V> attachment;
-	private final boolean isClient;
 	private final Map<ValueTarget, Object> map;
 
-	public AttachmentDictionary(Registry<R> registry, RegistryEntryAttachment<R, V> attachment, boolean isClient) {
+	public AttachmentDictionary(Registry<R> registry, RegistryEntryAttachment<R, V> attachment) {
 		this.registry = registry;
 		this.attachment = attachment;
-		this.isClient = isClient;
 		this.map = new HashMap<>();
 	}
 
 	public void put(Identifier id, Object value) {
-		this.map.put(new ValueTarget.Single(id), value);
+		this.map.put(new ValueTarget(id, ValueTarget.Type.ENTRY), value);
 	}
 
-	public void putTag(Identifier id, Object value, boolean required) {
-		this.map.put(new ValueTarget.Tagged<>(this.registry, id, this.isClient, required), value);
+	public void putTag(Identifier id, Object value) {
+		this.map.put(new ValueTarget(id, ValueTarget.Type.TAG), value);
```

Comment:
This allows you to set a value in a server/both-sided attachment to a client-side only tag (which will only work on a client, obviously).

Reply:
Ah, I see. Apparently vanilla does the same thing (for missing tags) - though it also logs that the tag is missing.